x/ibc/20-transfer/types/msgs.go

```diff
@@ -0,0 +1,63 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	TypeMsgTransfer = "transfer"
+)
+
+type MsgTransfer struct {
+	SrcPort      string         `json:"src_port" yaml:"src_port"`
+	SrcChannel   string         `json:"src_channel" yaml:"src_channel"`
+	Denomination string         `json:"denomination" yaml:"denomination"`
```

Comment:
we can use `sdk.Coins` instead of `Denomination` and `Amount`

Reply:
What about `sdk.Coin`?